02week/pigLatin.js

```diff
@@ -8,13 +8,85 @@ const rl = readline.createInterface({
 });
 
 
+const pigLatinMyString = (word, vowelPosition)=>{
+  if(vowelPosition == 0){
+    return word + "yay"
+  }
+  let mainString = word.split("");
+  const firstConsonants = mainString.splice(0,vowelPosition).join("");
+  //not sure if there is a single line way to do this, settled on this. Still thinking about it
+  mainString = mainString.join("") + firstConsonants + "ay"
```

Comment:
you're mutating this variable a lot here and it's unnecessary, join the array and return this. `return mainString.join("") + firstConsonants + "ay"`